### PR TITLE
Provider add token

### PR DESCRIPTION
**Pull request by haoyann**
Provider add token

### CODE REVIEW

**Review by xiaoheng1 — COMMENTED** [review]
please fix test case.

**Comment by haoyann** [comment]
> please fix test case.

Done.

File: dubbo-samples-zookeeper/src/main/resources/spring/dubbo-provider.properties [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #       http://www.apache.org/licenses/LICENSE-2.0
 #
 #   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
+#   distributed under the License is distributed on an "AS IS`" BASIS,
```
Comment (AlbumenJ) [inline dubbo-samples-zookeeper/src/main/resources/spring/dubbo-provider.properties:13]: please revocer this line

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-samples-zookeeper/src/main/resources/spring/dubbo-provider.properties (hunk 1 above)
Comment (haoyann) [reply dubbo-samples-zookeeper/src/main/resources/spring/dubbo-provider.properties]: Done.

**Review by xiaoheng1 — APPROVED** [review]
LGTM

**Review by AlbumenJ — APPROVED** [review]

**Pull request merged**